src/EngineManager.php

```diff
@@ -30,6 +31,14 @@ public function createAlgoliaDriver()
         ));
     }
 
+    public function createElasticsearchDriver()
+    {
+        return new Engines\ElasticsearchEngine(Elasticsearch::create()
```

Comment:
But you're hard coding the hosts config option. 
There is no way here for me to set additional options on the ES client.